src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -218,6 +219,21 @@ private void checkStringToIntWithBoundaries(
     }
   }
 
+  // version validation and migration logic goes here
+  // e.g. unsupported versions would fail or be migrated
+  // in this release we support only the current app version
+  private void validateTabulatorVersion() {
+    if (isNullOrBlank(getTabulatorVersion())) {
+      isValid = false;
+      Logger.log(Level.SEVERE, "Tabulator version is required!");
```

Comment:
```suggestion
      Logger.log(Level.SEVERE, "tabulatorVersion is required!");
```
Would you also mind changing the "Contest name" message to `contestName` while you're at it? These are the only two fields where the messages don't use the property names.